### ISSUE TITLE
Any way to make the output file zipped?

### ISSUE THREAD

**Issue by gonejack** [clue/phar-composer#104]
Using this command to make archives and find out that executable.phar is just a large plain text file without zipping.

```
php -d phar.readonly=off ./phar-composer.phar build . ./bin/executable.phar
```

**Comment by clue** [comment]
PRs are very much appreciated! :shipit: 

Compression make a whole lot of sense for many use cases. Though keep in mind that this means that both the creator and the user of the archive need to have `ext-zlib` enabled (which is not always available). Accordingly, this may limit how the resulting archive can be used. I'm fine with adding an option to control this, but this should probably default to being off.

As much as I'd love to work on this myself, there are currently no immediate plans to build this from my end (no demand at the moment and more important outstanding issues currently). If you need this for a commercial project and you want to help sponsor this feature, feel free to reach out and I'm happy to take a look.

**Comment by clue** [comment]
Closed via #3
